Jenkinsfile

```diff
@@ -98,7 +98,7 @@ def getFullBuild(jdk, os) {
                       //options: [invokerPublisher(disabled: false)],
                       mavenOpts: mavenOpts,
                       mavenLocalRepo: localRepo) {
-                sh "mvn -V -B install -Dmaven.test.failure.ignore=true -e -Pmongodb -T3 -DmavenHome=${mvntoolInvoker} -Dunix.socket.tmp="+env.JENKINS_HOME
+                sh "mvn -U -V -B install -Denv=ci -Dmaven.test.failure.ignore=true -e -Pmongodb -T3 -DmavenHome=${mvntoolInvoker} -Dunix.socket.tmp="+env.JENKINS_HOME
```

Comment:
Do we still need the `-U` option here?

Reply:
nope not using anymore surefire snapshot